### PR TITLE
Heat: Use Yaodu Ubuntu Image

### PR DESCRIPTION
**Pull request by intlabs**
This PR proposes the use of the Heat Image from the Yaodu Project.

For more information about the image please refer to:
 * https://hub.docker.com/r/yaodu/heat/
 * https://github.com/yaodu/docker-heat

---
This change is [Reviewable](https://reviewable.io/reviews/att-comdev/openstack-helm/116)

### CODE REVIEW

File: heat/values.yaml [hunk 1]
```diff
@@ -17,14 +17,14 @@ labels:
...
   db_init: quay.io/stackanetes/stackanetes-kolla-toolbox:newton
-  db_sync: docker.io/kolla/ubuntu-source-heat-api:3.0.1
-  ks_user: quay.io/stackanetes/stackanetes-kolla-toolbox:newton
-  ks_service: quay.io/stackanetes/stackanetes-kolla-toolbox:newton
-  ks_endpoints: quay.io/stackanetes/stackanetes-kolla-toolbox:newton
-  api: docker.io/kolla/ubuntu-source-heat-api:3.0.1
-  cfn: docker.io/kolla/ubuntu-source-heat-api:3.0.1
-  cloudwatch: docker.io/kolla/ubuntu-source-heat-api:3.0.1
-  engine: docker.io/kolla/ubuntu-source-heat-engine:3.0.1
+  db_sync: docker.io/yaodu/heat:ubuntu
```
Comment (alanmeadows) [inline heat/values.yaml]: Same comment here as in #115 - how does this allow openstack release targeting? I am assuming there is only one release of yaodu right now, but the naming seems short sighted?

File: heat/values.yaml [hunk 2]
```diff
@@ -108,7 +108,7 @@ resources:
     workers: 8
 
 misc:
-  debug: false
+  debug: true
```
Comment (alanmeadows) [inline heat/values.yaml]: We normally default to false.

File: heat/values.yaml (hunk 1 above)
Comment (SamYaple) [inline heat/values.yaml]: the naming follows [docker upstream](https://github.com/docker-library/official-images#tags-and-aliases) for better or worse.

Additionally, I have this https://etherpad.openstack.org/p/yaodu-tagging that is open to suggestions

**Comment by v1k0d3n** [comment]
i'd like to potentially use the slimmer yaodu images (default) as a complimentary project once in openstack proper, but i believe that we're going to let this PR drop (close) prior to the repo import.

cc: @intlabs @SamYaple

**Comment by intlabs** [comment]
@v1k0d3n that works for me - don't worry, I'll be back pushing these again when both projects are at a level of maturity that makes sense to do so :)

**Pull request closed without merging**